packages/widgets-mitosis/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+output
```

Comment:
You don't want to ignore all `output`. Rather, you just want to ignore `output/**/src` and allow everything else.

That's because you want `output/qwik/src/package.json` to be committed, along with `vite.config.js` and anything else needed to built the Mitosis output into a valid Qwik package